packages/plugins/i18n/admin/src/components/ModalEdit/BaseForm.js

```diff
@@ -19,10 +24,10 @@ const BaseForm = ({ locale }) => {
             id: getTrad('Settings.locales.modal.locales.label'),
             defaultMessage: 'Locales',
           })}
-          value={locale.code}
+          value={localeDetails.code}
           disabled
         >
-          <Option value={locale.code}>{locale.name}</Option>
+          <Option value={localeDetails.code}>{localeDetails.name}</Option>
```

Comment:
thank you for the update!
since localeDetails is undefined until isLoading becomes false, I think we should give it a fallback: 

```
<Select value={localeDetails?.code || locale.code}>
   <Option value={localeDetails?.code || locale.code}>{localeDetails?.name || locale.name}</Option>
</Select>
```

Reply:
hi, @ronronscelestes 
If there is no localeDetails?.name I think it should show code, otherwise it will be the same as before (show two same names, can't find code).